gulpfile.js

```diff
@@ -82,6 +83,68 @@ function createReloadServer() {
   });
 }
 
+function writeServiceWorkerFile() {
+  let path = require('path');
+  let rootDir = 'dist';
+
+  let config = {
+    cacheId: 'polymerproject',
+    staticFileGlobs: [
+      `${rootDir}/1.0/index.html`,
+      `${rootDir}/1.0/shell.html`,
+      `${rootDir}/1.0/about.html`,
+      `${rootDir}/elements/**`,
+      `${rootDir}/js/*.js`,
+      `${rootDir}/css/*.css`,
+      `${rootDir}/bower_components/**/webcomponents-lite.min.js`,
+    ],
+    dynamicUrlToDependencies: {
+      '/1.0/shell.html': [
+        path.join(rootDir, 'templates', 'base-devguide.html'),
```

Comment:
I don't see `base-devguide.html` (and `base-blog.html` or `page-md-styles.html`) used to uniquely determine the output of the `shell.html` template, so I think they can be left out.

Apologies if I'm missing some way that they're being used by sub-templates or something like that.